Overview.src.html

```diff
@@ -805,6 +823,15 @@ <h4 id=terminology-headers>Headers</h4>
 <span data-anolis-ref>REFERRER</span>
 
 <p>A <span title=concept-request>request</span> has an associated
+<dfn title=concept-request-client-hints-list>client hints list</dfn>, which is a
+<span title=concept-client-hints-list>client-hints list</span>. Unless stated otherwise, it is
+the empty list.
+
+<p class="note no-backref">This can be used to override a client hints list associated with
+an <span data-anolis-spec=html>environment settings object</span>.
+<span data-anolis-ref>CLIENT-HINTS</span>
```

Comment:
Right, that part hasn't been specced yet. My intent is to mirror similar language as [referrer policy](https://w3c.github.io/webappsec-referrer-policy/#terms).

- We can either do that within the CH draft.
- Or, we can integrate that language directly into the HTML spec?

Any preference on one vs other? /cc @mnot